cmd/handoff-bss-metadata.go

```diff
@@ -125,6 +143,10 @@ func init() {
 	handoffBSSMetadataCmd.Flags().StringVar(&csmVer, "csm-version",
 		"", "version of CSM that is currently running")
 	_ = handoffBSSMetadataCmd.MarkFlagRequired("data-file")
+	handoffBSSMetadataCmd.Flags().StringVar(&kubernetesFile, "kubernetes-file",
```

Comment:
I'm going to add another param for just specifying the version (e.g. `0.3.33`), and it can be used instead of the file.